### PR TITLE
don't prefix output properties check rule

### PR DESCRIPTION
**Pull request by eromano**
Added a simple rule for checking the name events without the prefix on:

https://angular.io/guide/styleguide#dont-prefix-output-properties

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -6,13 +6,13 @@ export {Rule as ContextualLifeCycleRule} from './contextualLifeCycleRule';
 export {Rule as DecoratorNotAllowedRule} from './decoratorNotAllowedRule';
 export {Rule as DirectiveClassSuffixRule} from './directiveClassSuffixRule';
 export {Rule as DirectiveSelectorRule} from './directiveSelectorRule';
-export {Rule as I18nRule} from './i18nRule';
```
Comment (mgechev) [inline src/index.ts]: Would you export this back?

**Comment by eromano** [comment]
done

**Comment by mgechev** [comment]
Cool! I'll include it in the 4.0.0 release.

**Comment by eromano** [comment]
Thanks

**Pull request merged**